codebuild/linux-gcc-4x-x86.yml

```diff
@@ -11,7 +11,7 @@ phases:
   build:
     commands:
       - echo Build started on `date`
-      - ./codebuild/common-linux.sh -DCMAKE_C_FLAGS="-m32" -DCMAKE_LINK_FLAGS="-m32"
+      - ./codebuild/common-linux.sh -DCMAKE_C_FLAGS="-m32" -DCMAKE_CXX_FLAGS="-m32" -DCMAKE_LINK_FLAGS="-m32"
```

Comment:
I think you need to export `CXX` on this one too, looks like it's using the builtin `/usr/bin/c++`.